.github/workflows/lintrunner_ci.yml

```diff
@@ -110,17 +110,13 @@ jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
-    if: "startsWith(github.ref, 'refs/tags/lintrunner/v*')"
     needs: [ macos, windows, linux, test ]
     steps:
-      - uses: actions/download-artifact@v2
```

Comment:
With this removed, what would be the step to download the lintrunner wheel uploaded by the previous build steps

Reply:
Generally the maturin publish command handles all of this for us, so we don't need to download the wheel.